<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>ATS 1054 Presentation</title>
    <meta charset="utf-8" />
    <meta name="author" content="Philip Leftwich" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <script src="libs/kePrint/kePrint.js"></script>
    <link href="libs/lightable/lightable.css" rel="stylesheet" />
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# ATS 1054 Presentation
## Non-Mendelian Inheritance
### Philip Leftwich
### 10th September 2021

---







class: center, middle

This session will be recorded for you to playback at any time, 
all slides are publically available on my GitHub page:

[Philip-Leftwich/nonmendelian_inheritance](https://github.com/Philip-Leftwich/nonmendelian_inheritance).


All diagrams and images are accompanied with alt-text.


---

# Hello! üëã

Let's warm up! 

There are two ways to engage today:

--

.pull-left[
## Teams Chat

- Text box for questions


![](images/nervous.gif)

]

--

.pull-right[
## Slido

- Go to slido.com

- Enter participant code 
### 733 633


]


---

## Learning Outcomes


1.  You will be able to `understand` why some crosses `don't` conform to Mendelian Inheritance

--

2.  You will be able to `analyze` genetic crosses for incomplete dominance &amp; co-dominance

--

3.  You will be able to `describe` gene interaction and `analyze` complex patterns of inheritance produced by epistasis

---


# Mendelian Inheritance

** a quick `recap`**

1. `Law of dominance`

&lt;img src="images/dominance.png" alt="Heterozygote shows dominant phenotype, recessive phenotype only visible in homozygotes"  style="width: 60%" /&gt;


---

# Mendelian Inheritance

** a quick `recap`**

1. `Law of dominance`

2. `Law of segregation` 

&lt;img src="images/segregation.png" alt="Diploid alleles separating in equal proportions into different gametes"  style="width: 60%" /&gt;

---

# Mendelian Inheritance

** a quick `recap`**

1. `Law of dominance`

2. `Law of segregation`

3. `Law of independent assortment` 

Genes of different traits sort independently. 


---

# A Dihybrid cross
** `peas` check out my Punnett square**

![Analysis of a dihybrid cross shows heterozygous plants are self-fertilised to produce an F2 generation with a 9:3:3:1 ratio](images/Slide1.png) 

---

# A Dihybrid cross
** `peas` check out my Punnett square**

![Analysis of a dihybrid cross shows heterozygous plants are self-fertilised to produce an F2 generation with a 9:3:3:1 ratio](images/Slide2.png)


---

# A Dihybrid cross
** `peas` check out my Punnett square**

![Analysis of a dihybrid cross shows heterozygous plants are self-fertilised to produce an F2 generation with a 9:3:3:1 ratio](images/Slide3.png)

---

# A Dihybrid cross
** `peas` check out my Punnett square**

![Analysis of a dihybrid cross shows heterozygous plants are self-fertilised to produce an F2 generation with a 9:3:3:1 ratio](images/Slide4.png)

---

# A Dihybrid cross
** `peas` check out my Punnett square**

![Analysis of a dihybrid cross shows heterozygous plants are self-fertilised to produce an F2 generation with a 9:3:3:1 ratio](images/Slide5.png)

---


# A Dihybrid cross


&lt;table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;"&gt;
 &lt;thead&gt;
&lt;tr&gt;&lt;th style="border-bottom:hidden;padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="4"&gt;&lt;div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; "&gt;F1 yellow, round x yellow, round&lt;/div&gt;&lt;/th&gt;&lt;/tr&gt;
  &lt;tr&gt;
   &lt;th style="text-align:left;"&gt; F2 &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; Of all offspring &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; Of all offspring &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; Combined probabilities &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:center;"&gt;  &lt;html&gt;&lt;body&gt;&lt;img src="images/arrow.png" width="16" height="16"&gt;&lt;/body&gt;&lt;/html&gt;
&lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 3/4 are round &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; (3/4)(3/4)=9/16 yellow, round &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 3/4 are yellow &lt;html&gt;&lt;body&gt;&lt;img src="images/blank.png" width="16" height="16"&gt;&lt;/body&gt;&lt;/html&gt;
&lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 1/4 are wrinkled &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; (3/4)(1/4)=3/16 yellow, wrinkled &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:center;"&gt;  &lt;html&gt;&lt;body&gt;&lt;img src="images/arrow.png" width="16" height="16"&gt;&lt;/body&gt;&lt;/html&gt;
&lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 3/4 are round &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; (1/4)(3/4)=3/16 green, round &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 1/4 are yellow &lt;html&gt;&lt;body&gt;&lt;img src="images/blank.png" width="16" height="16"&gt;&lt;/body&gt;&lt;/html&gt;
&lt;/td&gt;
   &lt;td style="text-align:center;"&gt; 1/4 are wrinkled &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; (1/4)(1/4)=3/16 green, wrinkled &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;

---

class: inverse, center, middle, 

## üß¨ Examples of `Non`-Mendelian Inheritance üß¨

.footnote[How do we interpret patterns of inheritance that seem to defy one or more of Mendel's Laws???]

---


## 1. `Partial` Dominance

Neither allele is dominant &amp; heterozygotes have an `intermediate` phenotype.

&lt;img src="images/incomplete.png" alt="Incomplete dominance shown in snapdragon plants, purebreeding red and white flowered plants cross to produce an intermediate pink flowered offspring"  style="width: 80%" /&gt; 

---

## 1. `Partial` Dominance

Partial dominance means the `phenotype` ratio is the **same** as the `genotype` ratio 

&lt;img src="images/cross.png" alt="Inbreeding of heterzygous pink flowers produces offspring in a 1:2:1 ratio of red:pink:white that is the same as the genotypes"  style="width: 50%" /&gt; 

---

## 2. `Co`dominance

Both alleles produce distinct, detectable gene products that are expressed `equally` in heterozygotes. 

&lt;table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;"&gt;
 &lt;thead&gt;
&lt;tr&gt;&lt;th style="border-bottom:hidden;padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2"&gt;&lt;div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; "&gt;The MN blood group&lt;/div&gt;&lt;/th&gt;&lt;/tr&gt;
  &lt;tr&gt;
   &lt;th style="text-align:left;"&gt; Genotype &lt;/th&gt;
   &lt;th style="text-align:center;"&gt; Phenotype &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; MM &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; M &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; MN &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; MN &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; NN &lt;/td&gt;
   &lt;td style="text-align:center;"&gt; N &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;
---

## 2. `Co`dominance

Both alleles produce distinct, detectable gene products that are expressed `equally` in heterozygotes. 

&lt;table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;"&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; MN X MN &lt;html&gt;&lt;body&gt;&lt;img src="images/blank.png" width="16" height="16"&gt;&lt;/body&gt;&lt;/html&gt;
&lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt;  &lt;html&gt;&lt;body&gt;&lt;img src="images/arrow.png" width="16" height="16"&gt;&lt;/body&gt;&lt;/html&gt;
&lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 1/4 MM &lt;html&gt;&lt;body&gt;&lt;img src="images/blank.png" width="16" height="16"&gt;&lt;/body&gt;&lt;/html&gt;
&lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 1/2 MN &lt;html&gt;&lt;body&gt;&lt;img src="images/blank.png" width="16" height="16"&gt;&lt;/body&gt;&lt;/html&gt;
&lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:center;"&gt; 1/4 NN &lt;html&gt;&lt;body&gt;&lt;img src="images/blank.png" width="16" height="16"&gt;&lt;/body&gt;&lt;/html&gt;
&lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;
---

## 2. `Co`dominance

Both alleles produce distinct, detectable gene products that are expressed `equally` in heterozygotes. 

&lt;img src="images/codominance.png" alt="something"  style="width: 75%" /&gt;


.footnote[*I* stands for isoagglutinogen, a `fancy` term for antigen]

---
class: middle, center, inverse

## ‚òÅüìä‚òÅ
# Question Time!

### [slido.com](slido.com)



### 733 633

---

## 3. Epistasis

`Gene interactions` - sometimes more than one gene is needed to produce a phenotype.

&lt;img src="images/Slide12.png" alt="the wildtype Fut1 allele directs conversion of a precursor to the H antigen, the IA and IB alleles are then able to convert the H antigen to either A or B allele"  style="width: 70%" /&gt;

---
## 3. Epistasis

`Gene interactions` - sometimes more than one gene is needed to produce a phenotype.

&lt;img src="images/Slide13.png" alt="the wildtype Fut1 allele directs conversion of a precursor to the H antigen, the IA and IB alleles are then able to convert the H antigen to either A or B allele"  style="width: 70%" /&gt;

---
## 3. Epistasis

`Gene interactions` - sometimes more than one gene is needed to produce a phenotype.

&lt;img src="images/Slide14.png" alt="the wildtype Fut1 allele directs conversion of a precursor to the H antigen, the IA and IB alleles are then able to convert the H antigen to either A or B allele"  style="width: 70%" /&gt;

---
## 3. Epistasis

`Gene interactions` - sometimes more than one gene is needed to produce a phenotype.

&lt;img src="images/Slide11.png" alt="the wildtype Fut1 allele directs conversion of a precursor to the H antigen, the IA and IB alleles are then able to convert the H antigen to either A or B allele"  style="width: 70%" /&gt;

---

## 3. Epistasis

The genotype of one gene `masks or modifies` the effect of another gene.

&lt;img src="images/epistasis.png" alt="a punnett square illustrating AA and BB crosses where both parents are heterzygotes for a recessive non-functional FUT1 allele. Offsrping of the type O blood phenotype are produced at a 3:1 ratio"  style="width: 80%" /&gt;

--

.footnote[Homozygotes for *fut1*&lt;sup&gt;h&lt;/sup&gt; always have an O blood phenotype]

---
class: middle, center

## ‚òÅüìä‚òÅ
### Question Time! [slido.com](slido.com) # 733 633
&lt;img src="images/final_3.png" alt="something" style="width: 90%" /&gt;

---
class: center, middle

# Exit Tickets üé´

Think üí≠ Pair üßëüèæ‚Äçü§ù‚Äçüßëüèº Share üì£

.pull-left[
"Which part of the lesson lost you today?"
![speech bubble one](images/speech.jpg)
]

.pull-right[
"Which part of the lesson do you understand well?"
![speech bubble two](images/speech_2.jpg)
]


---
class: center, middle

# Reading üìñ

Chapter 4. Extensions of Mendelian Genetics

Concepts of Genetics (10th or 11th  Edition); Klug W.S., Cummings M.R., Spencer C.A., Palladino M.A. 




---
class: center, middle

# Exercise üí™


Complete the quiz on Blackboard to test your knowledge 

Get 100% to earn your `Mendel Rules` Badge!!!

![Gregor Mendel Badge](images/mendel.png)



---

class: center, middle

# Thank you!


Web [https://philip-leftwich.github.io/](https://philip-leftwich.github.io/)


    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
